### PR TITLE
added note about clusters being removed

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-9065

### CODE REVIEW

File: observability/observability_enable.adoc [hunk 1]
```diff
@@ -36,7 +36,10 @@ Monitor the status of your managed clusters with the observability component, al
 
 Enable the observability service by creating a `MultiClusterObservability` custom resource instance. Before you enable observability, see xref:../observability/observe_environments.adoc#observability-pod-capacity-requests[Observability pod capacity requests] for more information. 
 
-*Note:* When observability is enabled or disabled on {ocp-short} managed clusters that are managed by {product-title-short}, the observability endpoint operator updates the `cluster-monitoring-config` ConfigMap by adding additional `alertmanager` configuration that automatically restarts the local Prometheus.
+*Note:* 
+
+- When observability is enabled or disabled on {ocp-short} managed clusters that are managed by {product-title-short}, the observability endpoint operator updates the `cluster-monitoring-config` ConfigMap by adding additional `alertmanager` configuration that automatically restarts the local Prometheus.
+- When you configure a setting in the {ocp-short} managed clusters, that cluster is removed.
```
Comment (jc-berger) [inline observability/observability_enable.adoc]: SMEs, why is the cluster removed? I think that explanation would really help here. 

Let me know why, and I'd be happy to include it here, .

**Review by berenss — APPROVED** [review]
lgtm thx! 🚀

File: observability/observability_enable.adoc [hunk 2]
```diff
@@ -36,7 +36,10 @@ Monitor the status of your managed clusters with the observability component, al
 
 Enable the observability service by creating a `MultiClusterObservability` custom resource instance. Before you enable observability, see xref:../observability/observe_environments.adoc#observability-pod-capacity-requests[Observability pod capacity requests] for more information. 
 
-*Note:* When observability is enabled or disabled on {ocp-short} managed clusters that are managed by {product-title-short}, the observability endpoint operator updates the `cluster-monitoring-config` ConfigMap by adding additional `alertmanager` configuration that automatically restarts the local Prometheus.
+*Note:* 
+
+- When observability is enabled or disabled on {ocp-short} managed clusters that are managed by {product-title-short}, the observability endpoint operator updates the `cluster-monitoring-config` ConfigMap by adding additional `alertmanager` configuration that automatically restarts the local Prometheus.
```
Comment (swopebe) [inline observability/observability_enable.adoc]: I know this isn't your entry, but config map here is an example of changes needed to fit IBM style. :-)

File: observability/observability_enable.adoc [hunk 3]
```diff
@@ -36,7 +36,10 @@ Monitor the status of your managed clusters with the observability component, al
...
+- The observability endpoint operator updates the `cluster-monitoring-config` ConfigMap by adding additional `alertmanager` configurations that automatically restart the local Prometheus. Therefore, when you insert the `alertmanager` configuration in the {ocp-short} managed cluster, the configuration removes the settings that relate to the retention of the Prometheus metrics.
```
Comment (swopebe) [inline observability/observability_enable.adoc]: same here.

**Review by swopebe — APPROVED** [review]
Thanks, Jake! Couple small comments. Approving.

**Pull request merged**